src/createPRContent.test.ts

```diff
@@ -15,7 +15,7 @@ This PR aggregates everything you've done in asssociated repos within the last w
 
 ### Next Steps for @${username}
 1. Review the PR
-    - If you notice missing information, add it in and ping @inkblotty to update the script
+    - If you notice missing information, add it in and ping \`@inkblotty\` to update the script
```

Comment:
Whole this alternative make sense:
```suggestion
    - If you notice missing information, add it in and create an issue in inkblotty/my-work-action
```